packages/desktop-gui/src/app/ui-blocker.scss

```diff
@@ -0,0 +1,8 @@
+.ui-blocker {
+  bottom: 0;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 99999;
```

Comment:
Another thing worth noting is that we should avoid explicitly setting the z-index; otherwise we will find ourselves fighting with out own code on who's on top.

As an alternative way to achieve the same goal, we can instead rely on the order of the portals created in the same root portal container (off of body). The order in which they are rendered is the same order in which they will appear in terms of their z-index. This also enables things like scrims on scrims; which is not an uncommon UI pattern with many major component libraries supporting this idea.